src/components/InstancesList/InstancesList.js

```diff
@@ -967,6 +968,7 @@ class InstancesList extends React.Component {
       if (isBrowseOption) {
         parentMutator.browseModeRecords.reset();
         this.setState({ isSingleResult: false });
+        goTo(path);
```

Comment:
I think it is necessary to leave the `qindex` param in the URL.

Reply:
Yes, you are right. Thank you for noticing that!